CONTRIBUTING.md

```diff
@@ -4,31 +4,51 @@ Thank you for contributing.
 
 See [Development](https://github.com/nvim-tree/nvim-tree.lua/wiki/Development) for environment setup, tips and tools.
 
+# Quality
+
+The following quality checks are mandatory and are performed during CI:
+
 ## Styling and formatting
 
 Code is formatted using luacheck, and linted using stylua.
-You can install these with:
+You may install these via your package manager or with:
 
 ```bash
 luarocks install luacheck
 cargo install stylua
 ```
 
+Run:
+```sh
+stylua lua
+luacheck lua
+```
+
 You can setup the git hooks by running `scripts/setup-hooks.sh`.
 
-## Adding new actions
+## Check
+
+[luals](https://luals.github.io) check is run with:
+
+```sh
+scripts/luals-check.sh
+```
+
+Requires `lua-language-server` on your path.
+
```

Comment:
Added to wiki as well: https://github.com/nvim-tree/nvim-tree.lua/wiki/Development#lua-language-server